packages/icons/src/index.js

```diff
@@ -165,6 +166,7 @@ export {
   Help,
   HelpSolid,
   Hopr,
+  IconRenderer,
```

Comment:
Merging this PR did not trigger a release of this package, a new release is needed to make the IconRenderer available through the public api when consuming this package via npm